cmd/pr-reminder/main.go

```diff
@@ -358,9 +358,18 @@ func main() {
 					logger := logrus.WithFields(logrus.Fields{
 						"kerberosId": user.KerberosId,
 					})
-					if err = sendMessage(logger, user.SlackId, user.PrRequests, slackClient); err != nil {
-						logger.WithError(err).Error("failed to message user")
-						errs = append(errs, err)
+
+					const maxPRsPerMessage = 40
```

Comment:
follow up of https://github.com/openshift/ci-tools/pull/4648/files#r2276151986, you won't end up making any change in the main function. You just need to add this logic to the sendMessage.